### PR TITLE
JS-data-model//MichaelCardona

### PR DESCRIPTION
**Pull request by MichaelCardona**
Screenshot 2021-04-01 210702

### CODE REVIEW

File: javascript-data-model/order-history.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+var orderHistory = [
+  {
+    shipTo: 'JS Masher',
+    orderNum: '114-3941689-8772232',
+    totalPrice: 34.00,
+    itemName: 'JavaScript for impatient programmer',
```
Comment (BrettAlbright) [inline javascript-data-model/order-history.js]: so the thing you want to add to this data model is a property `items` which will contain an array of objects in which each object contains an individual items data: 
```javascript
var orderHis = [
  {
    orderedDate: '2020-08-04',
    totalPrice: 34,
    shipTo: 'JS Masher',
    orderNum: '114-3941689-8772232',
    deliveredDate: '2020-08-08',
    orderedDate: '2-15-21',
    items:[{
      type: "book",
      title: "A New Way to Learn"
    },
    {
      type: "electronics",
      title: "Blu Ray Player"
    },
    {
      type: "book",
      title: "How to forget"
    },
    {
      type: "book",
      title: "how to remember"
    }
  ]
  }
]
```

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
great initial submission! small update requested!

File: javascript-data-model/order-history.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
+var orderHistory = [
+  {
+    shipTo: 'JS Masher',
+    orderNum: '114-3941689-8772232',
+    totalPrice: 34.00,
+    items: [{
```
Comment (BrettAlbright) [inline javascript-data-model/order-history.js]: almost there! just make sure to add the rest of the item data to the item object for all orders, author, item-price

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
almost there!

**Review by Zhiwutian — APPROVED** [review]
👍

**Pull request merged**